### ISSUE TITLE
Headless chicken melt into 144 mb of Molten Chicken, resulting in (after melting Chicken Head as well) total of 160 mb (10 nuggets) of Molten Chicken from single chicken, making it possible to infinitely produce chickens.

### ISSUE THREAD

**Issue by Lgmrszd** [quat1024/chickennugget#11]
Pls nerf.

**Comment by quat1024** [comment]
Lmao

**Comment by quat1024** [comment]
I don't think this is possible to fix. The headless chicken is not a new entity, it's just a vanilla chicken with a flag set, and I don't think the tinkers entity->fluid registry is that granular.

It's not *that* big a deal, because there's no way to automate head production, and you have to do this 9 times just to get one more chicken. You can already produce endless chicken from one in vanilla anyways (they lay eggs), and this is way harder than setting up an egg dispenser.

Thanks for making my day though lmao

**Comment by Lgmrszd** [comment]
Don't relax yet, I've found a couple of bugs saved up for later.

**Comment by quat1024** [comment]
Concern

**Comment by comp500** [comment]
~~yay more PRs for hacktoberfest~~
